PcAtChipsetPkg/PcatRealTimeClockRuntimeDxe/PcRtc.h

```diff
@@ -145,6 +145,24 @@ PcRtcInit (
   IN PC_RTC_MODULE_GLOBALS  *Global
   );
 
+/**

```

Comment:
I do not think this needs to be declare in PcRtc.h.  This function implemented in PcRtc.c and used in PcRtc.c.  Recommend moving implementation in PcRtc.c before first usage and no declaration required.